customize-direct-manipulation.php

```diff
@@ -54,6 +54,15 @@ public function init() {
 		add_filter( 'customize_widget_partial_refreshable', '__return_true', 20 );
 		add_filter( 'wp_page_menu_args', array( $this, 'maybe_add_page_menu_class' ) );
 		add_filter( 'wp_nav_menu_args', array( $this, 'make_nav_menus_discoverable' ) );
+
+		// Only Support certain theme / site types and blogs on business plan
+		// See plugins/footer-credit/customizer.php
+		if ( ! wpcom_is_vip_theme() && ! is_automattic() && ! wpcom_is_a8c_theme() ) {
```

Comment:
As per #25 we'll need to check for these functions before calling them since this plugin is available outside of wpcom. (You could probably just check for one of them and assume the rest will be there too.)